src/main/java/io/confluent/idesidecar/restapi/models/ConnectionSpec.java

```diff
@@ -236,9 +235,16 @@ public record CCloudConfig(
           description = "The identifier of the CCloud organization to use. "
                         + "The user's default organization is used when absent."
       )
-      @JsonProperty(value = "organization_id", required = true)
+      @JsonProperty(value = "organization_id")
       @Size(min = 36, max = 36)
-      String organizationId
+      String organizationId,
+      @Schema(
+          description = "The URI that users will be redirected to after successfully completing "
+          + "the authentication flow with Confluent Cloud."
+      )
+      @JsonProperty(value = "ide_auth_callback_uri")
+      @Size(min = 10, max = 200)
```

Comment:
I feel these are reasonable defaults for validating the `ide_auth_callback_uri`. It should have at least `len("vscode://") + 1` characters and is not expected to be longer than 200 characters.

Reply:
I see... I think I still have the question about why 200 in particular